Resources/Locale/en-US/interaction/interaction-popup-component.ftl

```diff
@@ -32,6 +33,7 @@ petting-failure-possum = You reach out to pet {THE($target)}, but are met with h
 petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodge with ludicrous speed!
 petting-failure-holo = You reach out to pet {THE($target)}, but {POSS-ADJ($target)} spikes almost impale your hand!
 petting-failure-dragon = You raise your hand, but as {THE($target)} roars, you decide you'd rather not be toasty carp food.
+petting-failure-hamster = You reach out to pet {THE($target)}, but {SUBJECT($target)} attempts to bite your finger, you quickly recoil your hand.
```

Comment:
```suggestion
petting-failure-hamster = You reach out to pet {THE($target)}, but {SUBJECT($target)} attempts to bite your finger, so you quickly recoil your hand.
```

Reply:
i will fix it although currently i do not think you can pet mice or hamster due to fact that when u click on a mouse u pick it up.

Reply:
i changed this line to the following.

"petting-failure-hamster = You reach out to pet {THE($target)}, but {SUBJECT($target)} attempts to bite your finger and only your quick reflexes save you from an almost fatal injury."

Would love to know your opinion on the change or if you have a suggestion for something better.

Reply:
sounds good